target/portfolio-1/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you delete your `target` folder and add a `.gitignore` that ignores `target/`? Since they're generated files from running `mvn package appengine:run`, they clog up source control a little bit and make it harder to review the relevant files :)

Reply:
Okay!!